client/components/notice/index.jsx

```diff
@@ -54,7 +54,13 @@ export const Notice = React.createClass( {
 		let content, text;
 
 		if ( typeof this.props.children === 'string' ) {
-			return <span className="notice__text"><span>{ this.props.children }</span></span>;
+			return (
+				<span className="notice__content">
+					<span className="notice__text">
+						<span>{ this.props.children }</span>
```

Comment:
Do we need the extra `span` wrapper? We don't have it in the other `.notice__text` below.